textsplitter/options.go

```diff
@@ -20,6 +23,7 @@ func DefaultOptions() Options {
 		ChunkSize:    _defaultTokenChunkSize,
 		ChunkOverlap: _defaultTokenChunkOverlap,
 		Separators:   []string{"\n\n", "\n", " ", ""},
+		LenFunc:      defaultLenFunc,
```

Comment:
this could just be `utf8.RuneCountInString` instead of being wrapped with defaultLenFunc, I believe.